torch/fx/graph_module.py

```diff
@@ -37,11 +37,14 @@ def patched_getline(*args, **kwargs):
     return _orig_getlines(*args, **kwargs)
 linecache.getlines = patched_getline
 
+
 def _forward_from_src(src: str, globals: Dict[str, Any]):
     # avoid mutating the passed in dict
     globals = globals.copy()
     exec_with_source(src, globals)
-    return globals['forward']
+    forward_fn = globals['forward']
+    del globals['forward']
```

Comment:
You can delete this line since you're making a local copy of `globals`

Reply:
I thought we want to delete 'forward' from `globals` here so that `GraphModule`'s generated `forward` won't have itself in its `__globals__` dict right? And dict copy is doing shallow copy so we would still have to delete `forward` here.